### ISSUE TITLE
[v7.8.1] Pipewire: volume is lower

### ISSUE THREAD

**Issue by Solarunit** [Taiko2k/Tauon#1217]
I am testing v7.8.1 with Pipewire and noticed that volume is lower when switching to it and restarting tauon.
(I don't change volume bar)

**Comment by C0rn3j** [comment]
Does the latest devel build/master suffer from the issue still?

https://github.com/Taiko2k/Tauon/wiki/Development-builds

**Comment by Solarunit** [comment]
I will test on next stable

**Comment by Solarunit** [comment]
v7.9.0 flatpak

Same issue. Volume on Pipewire is noticeably lower.

**Comment by C0rn3j** [comment]
Is this exclusive to Flatpak or reproducible outside of it too?

**Comment by Solarunit** [comment]
Only noticed this in tauon.

**Comment by C0rn3j** [comment]
Right, but is it limited to Tauon in Flatpak, or can you reproduce it in Tauon outside of Flatpak too?

You can try grabbing a CI build to test with, for example the one in the latest commit - https://github.com/Taiko2k/Tauon/actions/runs/12828659613

**Comment by Solarunit** [comment]
I downloaded this build. Tauon crashes when you switch to PipeWire. I can't test it.

PS: I checked current flatpak version on another system (Arch btw) and haven't noticed a difference.

offtopic: 
- why it archived 2 times - zip and 7z?
- also maybe use just `tauon` executable instead of `Tauon Music Box`

**Comment by C0rn3j** [comment]
Can you reproduce it on your current system but on a freshly new user?

Almost sounds like you have `~/.config/pipewire` present with some overrides but only for PW and not Pulse?

> why it archived 2 times - zip and 7z?

GitHub CI is awesome like that and zips everything.

> I downloaded this build. Tauon crashes when you switch to PipeWire. I can't test it.

Should be fixed by https://github.com/Taiko2k/Tauon/pull/1394, try https://github.com/Taiko2k/Tauon/actions/runs/12831468644?pr=1394

**Comment by Solarunit** [comment]
I think I fixed the issue. I removed: `.local/state/wireplumber` and restarted services 🎊
